source/configure/calls-deployment.md

```diff
@@ -185,13 +185,25 @@ RTCD is the only officially supported approach for Kubernetes deployments. For d
 
 ## When to Use RTCD
 
-The dedicated RTCD service (available with Enterprise license) is recommended for:
+This section will help you understand when and why your organization would want to use the dedicated RTCD service.
```

Comment:
@streamer45  - I believe this essentially restores all the previous content on "Why" that we had before?